### ISSUE TITLE
Add column for fixed vs infant-controlled trial length?

### ISSUE THREAD

**Issue by christinabergmann** [langcog/metalab2#101]
I think it's an interesting distinction that however might vary more across than within fields...

**Comment by juliacarbajal** [comment]
Good point, I think it's not a bad idea to add it. Quite related to this and to a previous issue on eye-tracking, I've been thinking that it would be important to make a distinction between online- and offline-coding.

**Comment by christinabergmann** [comment]
To update this, I propose the following:
column name: trial_control
levels:
fixed
infant_training (= infant-controlled in habituation/familiarization phase but *not* test)
infant_test (= infant-controlled in test phase but *not* in habituation/familiarization phase)
infant_all (all trials were infant-controlled, also when there was only a test-phase)

**Comment by shotsuji** [comment]
On this now with Cecile.
Decisions made:
- we add this variable for all studies, not only hab/fam. Thus, e.g. FC would be "infant_all", looking while listening would be "fixed"
- for hab/fam studies: We only consider a study infant controlled if the trial stops after infant looks away for x sec (thus, we do not consider it infant controlled because the sound starts playing once the infant looks)
